client/ayon_core/hosts/maya/api/lib.py

```diff
@@ -2525,33 +2525,39 @@ def get_fps_for_current_context():
 
 
 def get_frame_range(include_animation_range=False):
-    """Get the current folder frame range and handles.
+    """Get the current task frame range and handles.
 
     Args:
         include_animation_range (bool, optional): Whether to include
             `animationStart` and `animationEnd` keys to define the outer
             range of the timeline. It is excluded by default.
 
     Returns:
-        dict: Folder's expected frame range values.
+        dict: Task's expected frame range values.
 
     """
 
     # Set frame start/end
     project_name = get_current_project_name()
     folder_path = get_current_folder_path()
+    task_name = get_current_task_name()
+
     folder_entity = ayon_api.get_folder_by_path(project_name, folder_path)
```

Comment:
We only need the `id`
```suggestion
    folder_entity = ayon_api.get_folder_by_path(
        project_name, 
        folder_path,
        fields={"id"})
```